cultcargo/images/meqtrees/Dockerfile.kern9

```diff
@@ -0,0 +1,66 @@
+FROM kernsuite/base:9
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DEBIAN_PRIORITY=critical
+
+RUN docker-apt-install libblitz0-dev \
+    python3-dev \
+    python3-pip \
+    python3-venv \
+    python3-casacore \
+    casacore-dev \
+    casarest \
+    makems \
+    python-is-python3 \
+    wget \
+    git \
+    libblas-dev \
+    liblapack-dev \
+    cmake \
+    libblitz0-dev \
+    libqdbm-dev \
+    libfftw3-dev \
+    wcslib-dev \
+    libcfitsio-dev \
+    'libcasa-*' \
+    liblapack-dev \
+    libatlas-base-dev \
+    rsync
+
+
+
+RUN mkdir -p /usr/share/casacore/data/
+RUN rsync -avz rsync://casa-rsync.nrao.edu/casa-data /usr/share/casacore/data
+RUN pip install purr \
+    owlcat \
+    kittens \
+    'meqtrees-cattery>=1.7.6' \
+    astro-tigger-lsm \
+    python-casacore==3.5.2 \
+    numpy==1.21.5 \
+    --no-binary 'python-casacore' \
+    git+https://github.com/caracal-pipeline/cult-cargo@meqsim \
```

Comment:
Just a mental note, for this to keep working we should: (a) not delete the ``meqsim`` branch, (b) push out a 0.1.3rc1 release, and (c) change this to install cult-cargo>=0.1.3rc1.

All a bit of an awkward gymnastics routine, so maybe we should open an issue/discussion about how to handle this better (installing "self" into images), as I'm sure it will come up again...

Reply:
Ahh, of course. Let me convert this to a draft in the meantime

Reply:
OK but hang on, did you remove the cult-cargo install entirely? How does it even work without it?